src/index.js

```diff
@@ -1,3 +1,62 @@
-function printToConsole() {
-  console.log("Hello!");
+const heroText = [
+  "measuring traffic density_",
+  "understanding crop health_",
+  "tracking forest loss_",
+  "cell nuclei detection_",
+  "forecasting rip currents_",
+  "prediction of hourly global solar irradiation_",
+  "assessing storm damage_",
+];
+
+function changeHeroTextAndImage(moveForward) {
+  // find visible image id
+  const currentImgElem = document.getElementsByClassName("in-view")[0];
+  const currentIndex = parseInt(currentImgElem.id.split("-")[2]);
+  // restart from start or end and move left or right
+  const nextIndex =
```

Comment:
I think this would be a little easier to understand if it were done in multiple steps:
```javascript
const currentIndex = parseInt(currentImgElem.id.split("-")[2]); // No change here
const increment = moveForward ? 1 : -1;
const nextIndex = currentIndex + increment === heroText.length ? 0 : currentIndex + increment < 0 ? heroText.length - 1 : currentIndex + increment;
```

There should also be a "clever" solution using the mod operator that would allow this to happen via formula. I _think_ that something like this should do it (but haven't tested thoroughly):
```javascript
const currentIndex = parseInt(currentImgElem.id.split("-")[2]); // No change here
const increment = moveForward ? 1 : -1;
const nextIndex = (currentIndex + heroText.length) % heroText.length;
```

However, this isn't necessarily more readable and this isn't performance-critical code so either is fine. But in general, if you see a problem where you need to "wrap around" the ends of an array or something else that has a fixed length, the mod operator can usually do what you need.